docs/lsp.md

```diff
@@ -55,7 +53,7 @@ sequenceDiagram
     ```
 4. Uncomment the `__AMAZONQLSP_PATH` and `__AMAZONQLSP_UI` variables in the `amazonq/.vscode/launch.json` extension configuration
 5. Use the `Launch LSP with Debugging` configuration and set breakpoints in VSCode or the language server
-6. (Optional): Enable `"amazonq.trace.server": "on"` or `"amazonq.trace.server": "verbose"` in your VSCode settings to view detailed log messages sent to/from the language server. These log messages will show up in the "Amazon Q Language Server" output channel
+6. (Optional): Enable `"amazonq.lsp.trace": "on"` in your VSCode settings to view detailed log messages sent to/from the language server. These log messages will show up in the "Amazon Q Language Server" output channel. `"amazonq.lsp.logLevel"` can be used to configure the log level for the language server.
```

Comment:
It may be helpful to create a dedicated section to enabling logs here instead of as a single bullet point. Then you can link to it. It will also be useful to link to it from the CONTRIBUTING.md loggin section (search `### Logging`) as a subsection

Reply:
Since these settings are exposed to the user through the UI, I was treating those descriptions as the documentation. However, I agree that this is useful information to have in CONTRIBUTING.md so I've duplicated it there as where.